packages/mui-material/src/PaginationItem/PaginationItem.d.ts

```diff
@@ -14,23 +14,51 @@ export interface PaginationItemPropsSizeOverrides {}
 export interface PaginationItemPropsColorOverrides {}
 
 export interface PaginationItemSlots {
+  /**
+   * The component that renders the root slot.
```

Comment:
"root" slot is not there.
```suggestion
   * Component rendered for the "first page" button.
```